.circleci/config.yml

```diff
@@ -291,19 +332,33 @@ jobs:
           name: Convert test results
           command: |
             cd languages/csharp/AppEncryption
-            dotnet tool install -g trx2junit
-            export PATH="$PATH:/root/.dotnet/tools"
-            trx2junit AppEncryption.Tests/TestResults/*.trx
+            ~/.dotnet/tools/trx2junit AppEncryption.Tests/TestResults/*.trx
       - store_test_results:
           path: languages/csharp/AppEncryption/AppEncryption.Tests/TestResults
       - codecov/upload:
           file: languages/csharp/AppEncryption/AppEncryption.Tests/coverage.opencover.xml
           flags: languages_csharp_AppEncryption
+      - run:
+          name: Install locally
+          command: |
+            cd languages/csharp/AppEncryption
+            ./scripts/install_local.sh
+      - persist_to_workspace:
+          root: /root
+          paths:
+            - .nuget
   samples-csharp-reference-app:
     docker:
       - image: mcr.microsoft.com/dotnet/core/sdk:2.2
     steps:
       - checkout
+      - run:
+          name: Install tools
```

Comment:
minor (just now noticed): don't think this is needed for the sample app

Reply:
Should we also remove the deploy script then from the ReferenceApp?

Reply:
either way. doesn't matter as much. just trying to trim down the circleci config length